myia/validate.py

```diff
@@ -90,6 +90,7 @@ def validate_abstract(self, a: (type(None), AbstractExternal), uses):
     P.bool_and,
     P.bool_or,
     P.bool_eq,
+    P.stop_gradient,
```

Comment:
PlaceholderPrimitives should not be in the whitelist (if you rebase on top of current master, the whitelist was removed and replaced by a check for BackendPrimitive, so we won't need to keep updating this list anymore).

Reply:
Rebased!